bridge/device/cloud/publishResources.go

```diff
@@ -61,3 +87,51 @@ func (c *Manager) publishResources(ctx context.Context) error {
 	c.logger.Infof("resources published")
 	return nil
 }
+
+func getInstanceID(href string) int64 {
+	h := crc32.New(crc32.IEEETable)
+	_, _ = h.Write([]byte(href))
+	return int64(h.Sum32())
+}
+
+func toQuery(deviceID string, hrefs []string) string {
+	var buf strings.Builder
+	buf.WriteString("di=")
+	buf.WriteString(deviceID)
+	for _, href := range hrefs {
+		buf.WriteString("&ins=")
+		buf.WriteString(fmt.Sprintf("%v", getInstanceID(href)))
+	}
+	return buf.String()
+}
+
+func (c *Manager) unpublishResources(ctx context.Context) error {
+	firstRun := true
+	for {
+		// take only 10 resources to unpublish in one step because of the query length limit(255 characters)
+		readyResouces := c.popReadyToUnpublishResources(10)
```

Comment:
Isn't 10 too much in the worst case if the limit is 255?

"di=" (3 chars)
$deviceID = uuid in string format (36 chars)
"&ins=" (5 chars)
instanceID = h.Sum32 returns a 32 hash - max value is 4294967295 (10 digits)

== 54 chars in worst case

255 / 54 = 4,72, so we can pop max 4 per cycle to be sure